tests/ctst/steps/pra.ts

```diff
@@ -291,6 +291,22 @@ Then('the kafka DR volume exists', { timeout: 60000 }, async function (this: Zen
     assert(volumeParsed.result!['volume phase'] === 'Bound');
 });
 
+When ('I request the failover state for the DR', { timeout: 360000 }, async function (this: Zenko) {
+    await this.zenkoDrCtl?.failover({
+        sinkZenkoNamespace: 'default',
+        wait: true,
+        timeout: '6m',
```

Comment:
Should'nt we make the timeout a bit lower in the command than in the timeout for ctst ? so that we see the command fail before the step and we can have more logs..

Reply:
I'll update all functions yes

Reply:
Done -> https://github.com/scality/Zenko/pull/2144/commits/9208878fe2b628b1a9546ed9d7cffbce4414e650